thrift/src/main/resources/META-INF/plugin.xml

```diff
@@ -83,6 +83,8 @@
     <frameworkSupport implementation="com.intellij.plugins.thrift.config.facet.ThriftFacetSupportProvider"/>
     <facetType implementation="com.intellij.plugins.thrift.config.facet.ThriftFacetType"/>
     <compileServer.plugin classpath="thrift-jps.jar" />
+
+    <spellchecker.support language="thrift" implementationClass="com.intellij.spellchecker.JavaSpellcheckingStrategy"/>
```

Comment:
Hmm, was you able to confirm it locally?
You can run test build of a plugin with the `:thrift:runIde` command.
For me it doesn't seem to be working :thinking:

Reply:
I've confirmed that it worked locally, but it only works in comments right now, it's better to check the identifier name as well.

without the modification:
![image](https://github.com/user-attachments/assets/aa739cd0-e828-46ca-bc15-147955aca3e4)


with the modification:
![image](https://github.com/user-attachments/assets/64cb4373-2ef8-404c-9517-580f4a716c9a)

should we implement this functionality in a whole pr or make the rest in a following one?

Reply:
> should we implement this functionality in a whole pr or make the rest in a following one?

I see, if it's cover your use case up to some degree, I think we can proceed with incomplete implementation.


> Do you have time or interest in implementing this feature? I'm very willing to implement this feature myself,

Sure, I would be happy if you would work on this feature  and will support with prompt merge and release once it's ready :slightly_smiling_face:

Reply:
Thank you! I did some research this weekend and submitted a showcase file. Can you provide some advice?

Reply:
unit test cases added :)